### ISSUE TITLE
Removing dead legacy code from gaf6

### ISSUE THREAD

**Issue by Klazkin** [Klazkin/player-zero#29]
After #14 a lot of old bad code is still in the game (on Godot side in GD Script). This old bad code that makes it difficult to navigate the project and hinders further development. This will also help reassess what other improvements need to be further made.

## The goal

* [x] Reformat/Lint all existing .gd files.
* [x] Remove/refactor all of the classes that are marked as "Legacy".
* [x] Remove no longer needed comments, todo's etc.

## Time tracking

Time Estimate: `1 hours 30 minutes`
Time spent: `1 hours 13 minutes`

## Resources
* Linter https://pypi.org/project/gdtoolkit/
